### PR TITLE
Add the Web IDL Workstream and Standard

### PR DESCRIPTION
**Pull request by annevk**
See https://www.w3.org/2021/06/WHATWG-W3C-MOU_2021_update.html for context.

### CODE REVIEW

**Comment by annevk** [comment]
cc @TimothyGu @EdgarChen

**Review by TimothyGu — APPROVED** [review]

File: db.json [hunk 1]
```diff
@@ -464,6 +464,44 @@
...
+          "authors": [
+            {
+              "name": "Edgar Chen",
+              "email": "echen@mozilla.com"
+            },
+            {
+              "name": "Timothy Gu",
+              "email": "timothygu99@gmail.com"
+            }
+          ],
```
Comment (TimothyGu) [inline db.json:495]: Should we also add heycam to the list of authors here?

Comment (annevk) [reply db.json:495]: Last I checked @heycam was not interested in maintaining it. Once we get this merged we'll work with him on transferring the repository and putting redirects in place.

**Review by foolip — APPROVED** [review]

**Review by travisleithead — APPROVED** [review]
Approved!

**Review by othermaciej — APPROVED** [review]
Reviewing this in light of the requirements for a Workstream Proposal: https://whatwg.org/workstream-policy#workstream-proposals

Items 1-6 and 9-10 seem effectively fulfilled by either this PR or the issue it resolves. After walking through th Workstream Proposal requirements with the SG, we agreed that 7 and 8 do not seem reasonable to ask for up front in a proposal. I agree the waive those two requirements for this PR, and we will likely file an issue to remove them.

I also approve on the substance of the PR, and I believe WebKit would be interested in implementing this spec (and in fact already has a partial implementation).

**Comment by travisleithead** [comment]
From the WS proposal steps: https://whatwg.org/workstream-policy#workstream-proposals

Proposed Workstream name:
* Provided

Statement of purpose, including problem to be solved, proposed solution, status of the proposed solution, and how it is consistent with the WHATWG Principles and appropriate for a Workstream.
* Well established purpose

Specific Scope of work (included and excluded).
* Specified and noted.

Proposed deliverables.
* Noted in the JSON

Draft of the proposed Living Standard plus (optionally) a list of other proposed deliverables.
* Present: https://heycam.github.io/webidl/

A list of similar or related work being undertaken or proposed elsewhere and its relationship to the proposed Workstream.
* Used/referenced all over the web platform.

[waived] Anticipated commencement date and timeline for work going forward.
[waived] Anticipated Workstream Participants.

Description or list of those likely to implement or otherwise have an interest in the work.
* Chromium, Mozilla, WebKit interested in continuing to have this.

Expectations regarding long-term maintenance of the Living Standard.
* Existing editors planning to continue.

**Pull request merged**